_posts/en/newsletters/2024-06-28-newsletter.md

```diff
@@ -0,0 +1,125 @@
+  estimating the likelihood that an LN payment is feasible given the
+  public knowledge of a channel's maximum capacity but no knowledge
+  about its current balance distribution.  For example, Alice has a
+  channel with Bob and Bob has a channel with Carol.  Alice knows the
+  capacity of the Bob-Carol channel but not how much of
+  that balance is controlled by Bob and how much is controlled by Carol.
+
+  Pickhardt notes that some wealth distributions are impossible in a
+  payment network.  For example, Carol can't receive more money in her
+  channel with Bob than the capacity of that channel.  When all the
+  impossible distributions are excluded, it can be useful to consider
+  all the remaining wealth distributions as equally likely to occur.
+  That can be used to produce a metric for the likelihood that a payment
+  is feasible.
+
+  For example, if Alice wants to send a 1 BTC payment to Carol, and the
+  only channels it can pass through are Alice-Bob and Bob-Carol, then
+  we can look at what percentages of wealth distributions in the
+  Alice-Bob channel and the Bob-Carol channel would allow that payment
+  to succeed.  If the Alice-Bob channel has a capacity of several BTC,
+  most possible wealth distributions would allow the payment to succeed.
+  If the Bob-Carol channel has a capacity of just barely over 1 BTC, then
+  most possible wealth distributions would prevent the payment from
+  succeeding.  This can be used to calculate the overall likelihood
+  of the feasibility of a payment of 1 BTC from Alice to Carol.
+
+  The likelihood of feasibility makes it clear that many LN payments
+  that naively seem possible will not succeed in practice.  It also
+  provides a useful basis for making comparisons.
+  In a [reply][pickhardt feasible2], Pickhardt describes how the
+  likelihood metric could be used by wallets and business software to
+  automatically make some intelligent decisions on behalf of its users.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.18.1-beta][] is a minor release with a fix for "an [issue][lnd
+  #8862] that arises when handling an error after attempting to
+  broadcasting transactions if a btcd backend with an older version
```

Comment:
This is within a quote so may not be fixable?
```suggestion
  broadcast transactions if a btcd backend with an older version
```

Reply:
Made the edit.  I think it's fine.  I don't like editing if it's the quoted author's style choice or if there's any risk of changing the meaning, but small edits to fix clear typos are ok.